storage-resize-images/functions/src/index.ts

```diff
@@ -15,6 +15,7 @@
  */
 
 import * as admin from "firebase-admin";
+import { getEventarc } from "firebase-admin/lib/eventarc";
```

Comment:
I think this should probably be changed back to `import { getEventarc } from 'firebase-admin/eventarc';`

Logs currently:

```
ext-storage-resize-images-f7a8-generateResizedImage
Detailed stack trace: Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './lib/eventarc' is not defined by "exports" in /workspace/node_modules/firebase-admin/package.json
```

Reply:
Seeing the same error, so importing `lib/eventarc` doesn't work? @dackers86 you tested it and it worked, right?

Reply:
On the other PRs it was changed to `import { getEventarc } from 'firebase-admin/eventarc';` so I'm not sure 🤔 cc @dackers86

Reply:
@yamankatby just tested this internally with `import { getEventarc } from 'firebase-admin/eventarc';` and confirmed works

Reply:
Pushed a fix: https://github.com/firebase/extensions/pull/967/commits/044a3825c62503ad0878a4d82ce7025ae38c853a